src/Copy/BrandCopy.elm

```diff
@@ -447,88 +447,88 @@ brandCopy key =
 
         StoryOne1ImageAlt ->
             CopyText "✨"
 
         StoryOne1Message ->
-            CopyText ""
+            CopyText "They said my husband’s treatment had been appropriate but we knew the protocol for neutropenic sepsis."
 
         StoryOne2Quote ->
-            CopyText "✨"
+            CopyText "The leaflet about an advocate came with the reply. I had spoken to my husband. “What do you reckon, it’s a charity, we don’t have to but shall we give it a go?”. There were lots of forms but it was all very smooth. It was actually the week of my husband’s funeral when the worker called to arrange the first meeting, but I thought “I really do want to get on with this”."
 
         StoryOne2ImageAlt ->
             CopyText "✨"
 
         StoryOne2Message ->
-            CopyText "✨"
+            CopyText "You feel like you can’t say anything, it's the medical profession, they will close ranks against you."
 
         StoryOne3Quote ->
-            CopyText "✨"
+            CopyText "I had this real sense of relief that I was going to see somebody and when I met Susan, our advocate, it was so reassuring. She handed me a box of tissues and put together a letter in my name. Later, before the response meeting we met and she went through it with me, asking me which key things I wanted to remember to tell them."
 
         StoryOne3ImageAlt ->
             CopyText "✨"
 
         StoryOne3Message ->
-            CopyText ""
+            CopyText "You do worry that you might end up with a marker against your name."
 
         StoryOne4Quote ->
-            CopyText "✨"
+            CopyText "When we got to the response meeting it was amicable. They made big changes to the neutropenic sepsis process. Nurses can administer the intravenous drugs now, not just doctors and there are posters all over the hospital. I just felt like an enormous weight had been lifted - that’s all we’d wanted, them to admit that it wasn’t right."
 
         StoryOne4ImageAlt ->
             CopyText "✨"
 
         StoryOne4Message ->
-            CopyText "✨"
+            CopyText "I met someone who said “it’s thanks to you and your husband that I’m alive”."
 
         StoryTwoTitle ->
-            CopyText "✨"
+            CopyText "Lara"
 
         StoryTwoTeaser ->
-            CopyText "✨"
+            CopyText "Lots of younger people don’t make formal complaints. Lara’s story shows what could happen if they did."
 
         StoryTwoTeaserImageAlt ->
             CopyText "✨"
 
         StoryTwo1Quote ->
-            CopyText """✨"""
+            CopyText """I hate hospitals and I hate going to the doctor. I’ve had depression since I was a teenager and it took forever to get any treatment. So I didn’t want to go to A&E. But I had horrendous stomach cramps and I was throwing up, and I couldn’t get my temperature to go down and late one night, my friends convinced me to go."""
```

Comment:
The triple quotes are helpful for putting the code on multiple lines - which is also good for easier code review.

Reply:
want me to remove these? I mean, it's easier to read when reviewing code.

Reply:
The best would be to split them to multi-line which would make it easier to read - bunt since it's already in this way as a single line, probably not worth the effort.

Reply:
I can do it, no worries